app/src/main/res/layout/fragment_homepage.xml

```diff
@@ -13,17 +13,209 @@
+                android:textColor="@color/white"
+                android:textSize="17sp"
+                android:translationX="150dp"
+                android:translationY="40dp"
+                tools:ignore="HardcodedText,RtlHardcoded" />
+
+            <ImageView
+                android:id="@+id/Wolf_logo_info"
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                android:src="@drawable/pop"
+                android:translationX="10dp"
+                android:translationY="-65dp" />
+
+            <ImageView
+                android:id="@+id/Switch_off"
+                android:layout_width="154dp"
+                android:layout_height="87dp"
+                android:src="@drawable/switchoff"
+                android:translationX="240dp"
+                android:translationY="-65dp" />
+
+            <TextView
+                android:id="@+id/Switch_off_Info"
+                android:layout_width="233dp"
+                android:layout_height="91dp"
+                android:gravity="left"
+                android:text="When the switch is positioned to the left and lacks a purple color, the switch indicates the statement is false. "
+                android:textColor="@color/white"
+                android:textSize="17sp"
+                android:translationX="10dp"
+                android:translationY="-150dp"
+                tools:ignore="HardcodedText,RtlHardcoded" />
+
+            <TextView
+                android:id="@+id/Switch_on_info"
+                android:layout_width="244dp"
+                android:layout_height="82dp"
+                android:gravity="left"
+                android:text="When the switch is positioned to the right with a purple color, it signifies that the statement is true."
+                android:textColor="@color/white"
+                android:textSize="17sp"
+                android:translationX="150dp"
+                android:translationY="-140dp"
+                tools:ignore="HardcodedText,RtlHardcoded" />
+
+            <ImageView
+                android:id="@+id/Switch_on"
+                android:layout_width="125dp"
+                android:layout_height="wrap_content"
+                android:src="@drawable/switchon"
+                android:translationX="10dp"
+                android:translationY="-220dp" />
+
+            <ImageView
+                android:id="@+id/Plus"
+                android:layout_width="70dp"
+                android:layout_height="70dp"
+                android:src="@drawable/plus"
+                android:translationX="270dp"
+                android:translationY="-210dp" />
+
+            <TextView
+                android:id="@+id/Plus_info"
+                android:layout_width="248dp"
+                android:layout_height="63dp"
+                android:gravity="left"
+                android:text="Use the plus icon to indicate the ascending numerical value related to the statement."
```

Comment:
Here's another reminder to remember your target audience, because while fancy words can be funny, "the ascending numerical value related to the statement" isn't as straightforward as I know you could make it be, doesn't need to be fancy, just needs to get the point across